tests-remote/sshd_config

```diff
@@ -0,0 +1,41 @@
+#	$OpenBSD: sshd_config,v 1.102 2018/02/16 02:32:40 djm Exp $
+
+# This is the sshd server system-wide configuration file.  See
+# sshd_config(5) for more information.
+
+Port 2222
+
+# don't ever do this in real environments
+# this just lets us get around some permissions settings
```

Comment:
don't ever set this in a serious environment.
this allows us to bypass some permissions requirements.